web-browsers/the-dom-and-html.md

```diff
@@ -0,0 +1,122 @@
+| Element | A unit of content in an HTML document. These are represented as nodes in the DOM.                                                                                   | HTML Element              |
+| Sibling | In a tree structure, a sibling node is a node that shares a parent with another node                                                                                |
+
+### !callout-info
+
+## Web Pages Are Documents
+
+Some contexts may refer to a web page as a document.
+
+### !end-callout
+
+## The DOM
+
+The Document Object Model, also known as **the DOM**, is a representation of the structured content on a web page.
+
+The DOM is a _tree structure_. Each part of the document (or web page) is represented as a node in the tree.
+
+### Trees
+
+Trees are data structures which organize data through nodes and branches.
+
+There is one root node. The root node has children nodes.
+
+Those children nodes can also have their own child nodes! However, each node can either have exactly zero or one parent node.
+
+![A tree structure with grey, unlabeled nodes. The root node has two children. Each node has either 0, 1, or 2 children. The tree is four layers deep.](../assets/web-browsers_the-dom-and-html_generic-tree.png)
+
+When a web page is represented in the DOM structure, the root node of the tree is always a document object that represents the document itself.
+
+Elements and pieces of content become children nodes to the document.
+
+## HTML Elements Define Document Structure
+
+We organize the content of a webpage using _HTML_.
+
+**HTML**, or Hypertext Markup Language, is a language used to describe how the content is structured.
+
+Using HTML, we will turn every piece of content into **HTML elements**. Each _element_ of content will be organized in the document as a child and/or sibling node. How we structure our content in HTML will have a direct effect on the structure of the DOM.
+
+### Nested Elements
+
+When an element is nested inside another element, we represent this in the DOM by forming a parent-child relationship between the elements.
+
+### Sibling Elements
+
+When two elements share a parent node, they are considered to be _sibling elements_.
+
+## Example DOM and Relationships
+
+Consider the following example DOM. Each node is an HTML element with a name.
+
+![Diagram of a DOM that has a root node that is an `html` element. The following elements are present: html, head, title, meta, meta, body, h1, p, a, ul, li, li, li.](../assets/web-browsers_the-dom-and-html_dom-example.png)
+
+In this diagram, the `html` element is the root element.
+
+It has two children:
+
+1. A `head` element
+1. A `body` element
+
+The `head` and `body` elements are siblings with each other.
+
+The `head` element is contained inside the `html` element. It's a child of the `html` element. This `head` element itself, though, has three nested children elements:
```

Comment:
no adjective inflection
"though" sounds like it's making some sort of exception or contrast. we're just providing additional info.
```suggestion
The `head` element is contained inside the `html` element. It's a child of the `html` element. Additionally, the `head` element itself has three nested child elements:
```